src/execution/compiler/operator/analyze_translator.cpp

```diff
@@ -35,7 +35,6 @@ AnalyzeTranslator::AnalyzeTranslator(const planner::AnalyzePlanNode &plan, Compi
       num_rows_(GetCodeGen()->MakeFreshIdentifier("num_rows")),
       pg_statistic_index_iterator_(GetCodeGen()->MakeFreshIdentifier("pg_statistic_index_iterator")),
```

Comment:
This is also allocated and freed within the pipeline work. I think we probably have to move it to the pipeline state to avoid the same problem.